src/components/Input/Input.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { Story, Meta } from "@storybook/react";
+import Input from "./Input";
+import { InputProps } from "./type";
+
+import { ReactComponent as ArrowLeft } from "../../assets/arrow.svg";
+
+export default {
+  title: "Components/Input",
+  component: Input,
+} as Meta;
+
+// Create a master template for mapping args to render the Button component
+const Template: Story<InputProps> = (args) => <Input {...args} />;
+
+// Reuse that template for creating different stories
+export const DefaultInput = Template.bind({});
+DefaultInput.args = {
+  error: "tes",
+  help: "tes",
+  label: "Field label",
+  placeholder: "tes",
+  prefix: <ArrowLeft />,
+  suffix: "really long text",
+  disabled: false,
+} as InputProps;
```

Comment:
just wondering what will happen if `disabled: true` and error or help is not `null`

Reply:
kalo disabled, error dan help ga null, bakal tetep munculin errornya. 

gue ngikutin dari orbit sih:

https://kiwicom.github.io/orbit/?path=/story/inputfield--default-input

input mereka kalo error atau help ga null, tapi disabled, tetep bakal error gt

Reply:
hoo tp real usecase nya ga mungkin kejadian kan ya ? soalnya misal dia disabled + error berarti user ga bisa submit form sama ga bisa edit kan?